DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -292,16 +295,26 @@ function IsPackageInstalled
 }
 
 Function Test-LatestVersionInstalled {
+    [Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidUsingInvokeExpression','')]
     param(
         [Parameter(Position=0,Mandatory)]
-        [string]$pName
+        [string]$pName,
+        [Parameter(Position=1,Mandatory)]

```

Comment:
Yes, will make this change. Only added the positional parameter bindings because the other functions seemed to use them.